sim/target-agnostic.mk

```diff
@@ -176,7 +176,7 @@ $(PLATFORM) = $(OUTPUT_DIR)/$(DESIGN)-$(PLATFORM)
 $(PLATFORM): $($(PLATFORM))
 
 .PHONY: driver
-driver: $(PLATFORM)
+driver: $($(PLATFORM))
```

Comment:
What is this?  I don't see this in main at all.

Reply:
In the python, I change the build process to be "replace-rtl" and "build driver" instead of just "build driver" so I created a new rule called `driver` to do this instead of just calling the platform `f1` (which does everything). This makes it a bit easier to understand what is going on under the hood.